app-backend/app/src/main/scala/bucket/Routes.scala

```diff
@@ -95,6 +96,21 @@ trait BucketRoutes extends Authentication
                   }
                 }
               }
+            } ~
+            authenticate { user =>
+              post {
+                entity(as[BulkSceneOperation]) { bulkOp =>
+                  bulkOp.action match {
+                    case "upsert" => onSuccess(Buckets.addScenesToBucket(bulkOp.sceneIds, bucketId)) {
+                      _ => complete(StatusCodes.Created)
```

Comment:
The returned values are completely immaterial, since an `insertOrUpdate` if complete guarantees that all rows are there.